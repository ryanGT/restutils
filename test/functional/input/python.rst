.. py::

   x = 1

.. pyno::
   :echo: none

   from numpy import array,arange

.. py::

   x = array([[1]*10]*10)

.. py::

   x = arange(0,2,.1)
