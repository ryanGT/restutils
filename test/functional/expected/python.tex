% generated by Docutils <http://docutils.sourceforge.net/>
\documentclass[a4paper,english]{article}
\usepackage{fixltx2e} % LaTeX patches, \textsubscript
\usepackage{cmap} % fix search and cut-and-paste in PDF
\usepackage[T1]{fontenc}
\usepackage[latin1]{inputenc}
\usepackage{ifthen}
\usepackage{babel}
\usepackage{listings}
\usepackage{amsmath}

%%% Custom LaTeX preamble
% PDF Standard Fonts
\usepackage{mathptmx} % Times
\usepackage[scaled=.90]{helvet}
\usepackage{courier}

%%% User specified packages and stylesheets

%%% Fallback definitions for Docutils-specific commands

% hyperlinks:
\ifthenelse{\isundefined{\hypersetup}}{
  \usepackage[unicode,colorlinks=true,linkcolor=blue,urlcolor=blue]{hyperref}
  \urlstyle{same} % normal text font (alternatives: tt, rm, sf)
}{}

%%% Body
\begin{document}
\begin{equation}\begin{split}
x  = 1
\end{split}\end{equation}


\begin{lstlisting}[language={python}]

\end{lstlisting}
\begin{equation}\begin{split}
x  = \begin{bmatrix}
1 & 1 & 1 & 1 & 1 & 1 & 1 & 1 & 1 & 1
\\
1 & 1 & 1 & 1 & 1 & 1 & 1 & 1 & 1 & 1
\\
1 & 1 & 1 & 1 & 1 & 1 & 1 & 1 & 1 & 1
\\
1 & 1 & 1 & 1 & 1 & 1 & 1 & 1 & 1 & 1
\\
1 & 1 & 1 & 1 & 1 & 1 & 1 & 1 & 1 & 1
\\
1 & 1 & 1 & 1 & 1 & 1 & 1 & 1 & 1 & 1
\\
1 & 1 & 1 & 1 & 1 & 1 & 1 & 1 & 1 & 1
\\
1 & 1 & 1 & 1 & 1 & 1 & 1 & 1 & 1 & 1
\\
1 & 1 & 1 & 1 & 1 & 1 & 1 & 1 & 1 & 1
\\
1 & 1 & 1 & 1 & 1 & 1 & 1 & 1 & 1 & 1
\end{bmatrix}
\end{split}\end{equation}
\begin{equation}\begin{split}
x  = \left[ \begin{array}{ccccc}
0 & 0.1 & 0.2 & 0.3 & 0.4
\end{array} \right.  \ldots \\
\left. \begin{array}{ccccc}
1.5 & 1.6 & 1.7 & 1.8 & 1.9
\end{array} \right]
\end{split}\end{equation}

\end{document}
